libs/barista-components/expandable-panel/src/expandable-panel.ts

```diff
@@ -71,6 +77,9 @@ import { Observable } from 'rxjs';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class DtExpandablePanel {
+  /** @internal Unique identifier on the page for the expandable panel */
+  _uniqueId = `dt-expandable-panel-${uniqueId++}`;
+
   /** Whether the panel is expanded. */
```

Comment:
We should add an Input for the id on the component here and use the id if given otherwise fallback to the uniqueid

Reply:
I agree on the input part, I will take care of this.